src/main/kotlin/me/owdding/skyblockpv/feature/ProfileViewerButton.kt

```diff
@@ -0,0 +1,45 @@
+package me.owdding.skyblockpv.feature
+
+import me.owdding.ktmodules.Module
+import me.owdding.skyblockpv.utils.Utils
+import net.minecraft.world.item.Items
+import tech.thatgravyboat.skyblockapi.api.events.base.Subscription
+import tech.thatgravyboat.skyblockapi.api.events.screen.InventoryChangeEvent
+import tech.thatgravyboat.skyblockapi.api.item.replaceVisually
+import tech.thatgravyboat.skyblockapi.utils.regex.RegexUtils.match
+import tech.thatgravyboat.skyblockapi.utils.text.Text
+import tech.thatgravyboat.skyblockapi.utils.text.TextColor
+import tech.thatgravyboat.skyblockapi.utils.text.TextStyle.color
+
+@Module
+object ProfileViewerButton {
+
+    private val titleRegex = "(?<name>.*)'s Profile".toRegex()
+
+    @Subscription
+    fun onInventoryChange(event: InventoryChangeEvent) {
+        if (event.slot.index != 13) return
```

Comment:
A check for if an actual item is already there would be good as to not cause issues if hypixel updates